src/scss/underline.scss

```diff
@@ -12,7 +12,7 @@
     padding-right: 0px;
     //background: none repeat scroll 0 0 transparent;
     background: $color;
-    margin-top: 5px;
+    transform: translateY(5px);
```

Comment:
margin-top was causing items w/ underline functionality to have 2px of extra spacing on the bottom, causing vertical alignment issues. this transform implements the same relative offset for the underline without growing the element by 2px